ocs_ci/framework/__init__.py

```diff
@@ -352,5 +352,59 @@ def is_consumer_exist(self):
         cluster_types = [cluster.ENV_DATA["cluster_type"] for cluster in self.clusters]
         return "consumer" in cluster_types
 
+    def is_cluster_type_exist(self, cluster_type):
+        """
+        Check if the given cluster type exists in the clusters
+
+        Args:
+            cluster_type (str): The cluster type
+
+        Returns:
+            bool: True, if the given cluster type exists in the clusters. False, otherwise.
+
+        """
+        cluster_types = [cluster.ENV_DATA["cluster_type"] for cluster in self.clusters]
+        return cluster_type in cluster_types
+
+    def get_cluster_type_indexes_list(self, cluster_type):
+        """
+        Get the cluster type indexes
+
+        Returns:
+            list: the cluster type indexes
+
+        Raises:
+            ClusterNotFoundException: In case it didn't find any cluster with the cluster type
+
+        """
+        cluster_type_indexes_list = []
+        for i, cluster in enumerate(self.clusters):
+            if cluster.ENV_DATA["cluster_type"] == cluster_type:
```

Comment:
Should we rather use get here?

Reply:
Thanks for revising the pr. From what I know, the param `cluster_type` should always exist in the run. And if not provided, the default value is empty string `''`.